Components does not declare dependency on rich-text.

The `autocomplete` component in `@wordpress/components` uses `@wordpress/rich-text` but this dependency isn't included in the package's `dependencies`. 

```
[Error] Error: Cannot find module '@wordpress/rich-text'
	webpackMissingModule (dashboard.js:2712:138)
	./node_modules/@wordpress/components/build-module/autocomplete/index.js (dashboard.js:2712:141)
	__webpack_require__ (dashboard.js:20)
	./node_modules/@wordpress/components/build-module/index.js (dashboard.js:11284:91)
	__webpack_require__ (dashboard.js:20)
	./pro/dashboard/entries/dashboard/app.js (dashboard.js:118727:100)
	__webpack_require__ (dashboard.js:20)
	./pro/dashboard/entries/dashboard.js (dashboard.js:118671:95)
	__webpack_require__ (dashboard.js:20)
	(anonymous function) (dashboard.js:84)
	Global Code (dashboard.js:85)
```

--- a/packages/components/package.json
+++ b/packages/components/package.json
@@ -31,6 +31,7 @@
 		"@wordpress/i18n": "file:../i18n",
 		"@wordpress/is-shallow-equal": "file:../is-shallow-equal",
 		"@wordpress/keycodes": "file:../keycodes",
+		"@wordpress/rich-text": "file:../rich-text",
 		"@wordpress/url": "file:../url",
 		"classnames": "^2.2.5",
 		"clipboard": "^2.0.1",

